client/Windows/wf_event.c

```diff
@@ -567,7 +567,7 @@ LRESULT CALLBACK wf_event_proc(HWND hWnd, UINT Msg, WPARAM wParam,
 			break;
 
 		case WM_SETCURSOR:
-			if (LOWORD(lParam) == HTCLIENT)
+			if (wfc && LOWORD(lParam) == HTCLIENT)
```

Comment:
Sure the else branch is still correct?

Reply:
I was just trying to avoid the exception, but in looking at this, the WM_SETCURSOR case where wfc != NULL has already been handled above, so this will never do anything. Plus hDefaultCursor is never referenced anywhere else, and even if it was, we wouldn't be able to access it here when wfc is NULL so I'll just take out this entire case statement